CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [2.12.5] - 2020-09-08
 
+- Added background under onSurface ([#146](https://github.com/Shopify/polaris-tokens/pull/146))
```

Comment:
This should go under unreleased. We can cut a new release tomorrow or Thursday together if you want